examples/2016-10-31/lambda_edge/template.yaml

```diff
@@ -61,24 +61,14 @@ Resources:
           Handler: index.handler
           Timeout: 5
           # More info at https://github.com/awslabs/serverless-application-model/blob/master/docs/safe_lambda_deployments.rst
-          AutoPublishAlias: live 
-
-    LambdaEdgeFunctionRole:
-      Type: "AWS::IAM::Role"
-      Properties:
-          Path: "/"
-          ManagedPolicyArns:
-              - "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
+          AutoPublishAlias: live
           AssumeRolePolicyDocument:
```

Comment:
@timoschilling Thanks for the contribution! Is there a reason why this is not supported in globals? To make that change you would need to add it [here](https://github.com/awslabs/serverless-application-model/blob/develop/samtranslator/plugins/globals/globals.py#L40)

Reply:
I never think about that. I will add it to globals

Reply:
done